sematic/cli/run.py

```diff
@@ -88,6 +88,7 @@ def run(build: bool, log_level: str, script_path: str, script_arguments: Tuple[s
     # ensure the server is reachable before doing all the work
     validate_server_compatibility(use_cached=False)
 
+    run_command = " ".join(sys.argv)
```

Comment:
Should we store it as a list of string instead of a string?

It may make formatting easier in the dashboard.

Reply:
[PostgreSQL](https://www.postgresql.org/docs/current/arrays.html) and [SQLite](https://stackoverflow.com/questions/9755741/vectors-lists-in-sqlite) array types seems to be different (SQLite does not have native support, so we must simulate it via `JSON`).

If it really comes to it, instead I would rather split by space in JavaScript, taking care not to split within quotes.

Reply:
cc @chance-an